pyrokinetics/local_geometry/LocalGeometryMiller.py

```diff
@@ -1,10 +1,33 @@
+import numpy as np
+from typing import Tuple, Dict, Any
 from scipy.optimize import least_squares  # type: ignore
 from ..constants import pi
-import numpy as np
 from .LocalGeometry import LocalGeometry
 from ..equilibrium import Equilibrium
 from ..typing import Scalar, ArrayLike
-from typing import Tuple
+
+
+def get_default_miller_inputs():
```

Comment:
Just a little bike-shedding:

```suggestion
def default_miller_inputs():
```

although even better would be to have these as default argument to `LocalGeometryMiller.__init__` -- that's for another PR though